locales/en/index.yml

```diff
@@ -2899,6 +2899,8 @@ bonus:
       toast:
         ok: "Voucher saved successfully"
         ko: "Problems saving the voucher"
+  cdc:
+    title: "Carta della cultura"
```

Comment:
❓ This should be translated in eng, or it's a temporary placeholder?

Reply:
AFAIK the name of the bonus is the same both in Italian and in English.
But I added a task to check the texts before the release.